### DISCUSSION TITLE
Translator's Questions

### DISCUSSION THREAD

**Discussion by uli-on** [TeamAmaze/AmazeFileManager#3544] [Q&A]
Translating both terms, **_Appearance_** and **_Interface_** with **_Aussehen_** leads to confusion. Interface is **_Benutzeroberfläche_**, and if too long can be shortened to **_Nutzeroberfläche_**, or simply **_Schnittstelle_**. Not as speaking as my first proposal but better than the current term that doesn't let users differentiate.

**Comment by uli-on** [comment]
There's this menu item "Installieren" (English origin is probably "install", in the Quick Access section). It opens an endless list of images, though. Is this translation correct/misplaced, or can it be improved? How?

How to proceed with further findings, post here and rename the topic? Open new issues for each? Contact the translator?

**Comment by TranceLove** [comment]
@uli-on You can help improve the translation via transifex.

https://www.transifex.com/amaze/amaze-file-manager

**Comment by uli-on** [comment]
Is the German translator active still? (Conflicts foreseeable in that case.)

Remaining:
> Is this translation correct/misplaced, or can it be improved? How?

**Comment by TranceLove** [comment]
@uli-on Not sure if the original translator is still actively working on it, but I think you can still provide your suggestions. If there is no opposition we may integrate them into our codebase.

**Comment by uli-on** [comment]
Thanks. 

But please: Re the - in my opinion - wrong string for installieren/install, there's no fix/proposal/opinion/hint, yet

**Comment by VishnuSanal** [comment]
> But please: Re the - in my opinion - wrong string for installieren/install, there's no fix/proposal/opinion/hint, yet

@EmmanuelMess

**Comment by EmmanuelMess** [comment]
No one should use the same word twice, you should change the current translation.

**Comment by uli-on** [comment]
Guys, I've just registered to Transifex, logged in and tried, and it's a true mess, this software. Never seen seen such b...t! I'll never again blame s.o. for faulty translations. Honestly!

**Comment by VishalNehra** [comment]
Is there an issue? Are you not able to translate there?

**Comment by uli-on** [comment]
Not on a Mobile (no mobile theme, 25% font size), not with a black theme (not editable). I would have had to shift the page around which lead to incidentally tapping wrong fields/strings... and registering my mail cost 6 open browser windows.
But I'm now at my desktop PC and it's better, I'll find my way. I'm used to Crowdin, sorry.

**Comment by uli-on** [comment]
OK, I've also finished all open strings now, except for one (ftp over ssl connection) where I've asked a technical question.

**Comment by uli-on** [comment]
I'd like to point to something that looks like a structural/organisational problem, see screenshot from Transifex. It's about the translation of "Images", but similar might have happened elsewhere and in 77 languages. Please read from bottom up. 

![Amaze Translation German Images History](https://user-images.githubusercontent.com/35634829/192167590-18fb5d11-967c-4374-94e7-5b1a859432f1.jpg)

"Bilder" is the *correct* German term for "Images".
"Installieren" is the German term for the verb "install", BTW.

**Comment by uli-on** [comment]
Heads up, guys, two questions in one post!

1 - How should I handle NEW questions re translation, ask here (after renaming the topic to e.g. "Translator's Questions") or open a new topic each time? I'm asking because:

Over at xda there's been a question that points to a translation issue in German language. 

2 - Hence: will the menu width (one file selected > three dots menu) grow automatically or will I have to abbreviate?

**Comment by VishnuSanal** [comment]
@uli-on thanks for your concern!

> 1 - How should I handle NEW questions re translation, ask here (after renaming the topic to e.g. "Translator's Questions") or open a new topic each time? I'm asking because:

you can put them in here, I'll do the rename.

> 2 - Hence: will the menu width (one file selected > three dots menu) grow automatically or will I have to abbreviate?

it will automatically grow till a certain limit. the text will be ellipsized at the end after that. please see the attached image for reference.

<img src="https://user-images.githubusercontent.com/50027064/193106260-aee5cfa6-44b9-4981-8041-51a9d0385496.jpg" width=400/>

**Comment by uli-on** [comment]
Yesss, that overly long one was exactly the item in question on xda. What I see in the screenie should suffice even for German compounds (Thanks, Vishnu!)

**Comment by uli-on** [comment]
OK, here's one I had in mind where I needed assistance: 

## NEW TOPIC IS "TASK" (Open Files As New Task)

There are several strings mentioning "task". In German it's one time been used literally ("Task" as a technical term exists in German), another time translated to "Aufgabe" (which is "task for a person"). Personally I never seem to have encountered a " task", so I need to ask: Wikipedia equals it to process, which isn't speaking enough for me to become smarter. Is it about something like "open in separate window"? Which it doesn't do either.

**Comment by EmmanuelMess** [comment]
> There are several strings mentioning "task". In German it's one time been used literally ("Task" as a technical term exists in German), another time translated to "Aufgabe" (which is "task for a person"). Personally I never seem to have encountered a " task", so I need to ask: Wikipedia equals it to process, which isn't speaking enough for me to become smarter. Is it about something like "open in separate window"? Which it doesn't do either.

In this case "task" refers to specific thing in the Android OS, read about it here: https://developer.android.com/guide/components/activities/tasks-and-back-stack

**Comment by uli-on** [comment]
Thanks, Emmanuel, but that article somehow trapped me like in a sparrow net ;) too far away from practise.

I'll ask differently: What is the advantage of using this setting, and what might be the disadvantages so that users are allowed to turn it off? In my tests on all kinds of file formats I couldn't find differences in behaviour at all.

**Comment by VishnuSanal** [comment]
> I'll ask differently: What is the advantage of using this setting, and what might be the disadvantages so that users are allowed to turn it off? In my tests on all kinds of file formats I couldn't find differences in behaviour at all.

related #2766 #2783

**Comment by uli-on** [comment]
Ah, that explains both the feature and the situation! Thanks, Vishnu
